### PR TITLE
Symfony security

### PR DESCRIPTION
**Pull request by ntsanq**

### CODE REVIEW

File: src/Controller/Admin/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    #[IsGranted('ROLE_ADMIN')]
+    #[Route('/admin', name:'app_admin')]
+    public function index(): Response
+    {
+        $user = $this->getUser();
+        return $this->render('admin/index.html.twig',[
+            'admin'=>$user
+        ]);
+    }
+}
```
Comment (richardktran) [inline src/Controller/Admin/AdminController.php:21]: EOL

File: src/Controller/LoginController.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        if ($this->isGranted('ROLE_ADMIN')){
+           return $this->redirectToRoute('app_admin');
+        }
+        return $this->redirectToRoute('app_profile');
+    }
+
+    #[Route('/logout', name: 'logout')]
+    public function logout()
+    {
+        var_dump('log out success');die;
```
Comment (khajackie2206) [inline src/Controller/LoginController.php:39]: die??

File: src/Controller/User/ProfileController.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    #[IsGranted('ROLE_USER')]
+    #[Route('/profile', name:'app_profile')]
+    public function index(): Response
+    {
+        $user = $this->getUser();
+        return $this->render('user/index.html.twig',[
+            'user'=>$user
+        ]);
+    }
+}
```
Comment (tringuyennfq) [inline src/Controller/User/ProfileController.php:22]: EOL bro

File: templates/login/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+        <label for="password">Password:</label>
+        <input type="password" id="password" name="_password"/>
+
+        {# If you want to control the URL the user is redirected to on success
+        <input type="hidden" name="_target_path" value="/account"/> #}
+
+        <button type="submit">login</button>
+    </form>
+{% endblock %}
```
Comment (tringuyennfq) [inline templates/login/index.html.twig:22]: another EOL

File: src/Controller/User/ProfileController.php (hunk 3 above)
Comment (khajackie2206) [inline src/Controller/User/ProfileController.php:22]: EOL

File: templates/user/index.html.twig [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Welcome USER: {{ user.username }}
+<form action="{{ path('logout') }}" method="post" id="form-button-logout">
+    <input type="hidden" name="_csrf_token" value="{{ csrf_token('logout') }}" id="csrf-token"/>
+    <input
+            class="btn btn-danger border-0 rounded-0 text-white" type="submit" id="_submit"
+            name="_submit"
+            value="Logout"
+    />
+</form>
```
Comment (khajackie2206) [inline templates/user/index.html.twig:9]: EOL

File: src/Controller/LoginController.php [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        // last username entered by the user
+        $lastUsername = $authenticationUtils->getLastUsername();
+
+        return $this->render('login/index.html.twig', [
+            'last_username' => $lastUsername,
+            'error' => $error,
+        ]);
+    }
+
+    #[Route('/login_success', name: 'login_success')]
```
Comment (bangdinhnfq) [inline src/Controller/LoginController.php:27]: No need /login_success. Just redirect to profile page

Comment (ntsanq) [reply src/Controller/LoginController.php:27]: thank you. I'll fix it

File: src/Controller/LoginController.php (hunk 2 above)
Comment (ntsanq) [reply src/Controller/LoginController.php:39]: my bad, I create the logging out function in security.yaml

Comment (ntsanq) [reply src/Controller/LoginController.php:39]: my bad

**Pull request merged**